tools/cprnc/README

```diff
@@ -12,30 +12,19 @@ your case directory.
 Quick Start Guide:
 ------------------
 
-On cime supported systems you can generate a Macros file using the following
+On cime supported systems you can generate a cmake Macros file using the following
 (assuming you are running the command from the directory cime/tools/cprnc):
 
-CIMEROOT=../.. ../configure --macros-format=Makefile --mpilib=mpi-serial
```

Comment:
We want users to use CMake. Unfortunately, there is still a Makefile build of cprnc in `tools/mapping/gen_domain_files/test_gen_domain.sh`, so I couldn't get rid of the Makefile.